xhr.bs

```diff
@@ -1008,8 +1015,13 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
  <a event><code>abort</code></a>, and "{{AbortError!!exception}}" {{DOMException}}.
 
  <li><p>Otherwise, if <var>xhr</var>'s <a for=XMLHttpRequest>response</a>'s is a
- <a for=/>network error</a>, run the <a>request error steps</a> for <var>xhr</var>,
- <a event><code>error</code></a>, and "{{NetworkError!!exception}}" {{DOMException}}.
+ <a for=/>network error</a>, then perform the following steps:
+ <ol>
+  <li><p><a for=/>Report timing</a> for <var>xhr</var>.
```

Comment:
I think this has to happen in the request error steps, no? Otherwise how would we report for a timeout or abort scenario?